simple_scrape.py

```diff
@@ -171,15 +183,27 @@ def process_article(content, article, meta):
 
     if len(uninteresting_sentence_params) > 0:
         sentence_params = random.sample(uninteresting_sentence_params, 1)[0]
-        post(**sentence_params)
+        word = sentence_params.get("word").lower()
+        if word and get_uninteresting_count_for_word(word) < 1000:
+            post(**sentence_params)
+
+            md5_fn.update(sentence_params.get("sentence").encode("utf-8"))
+            sentence_hash = md5_fn.hexdigest()
```

Comment:
The md5 hash stuff is no longer necessary and has been removed in the `batch-3` branch.